src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -67,39 +80,23 @@ class Config:
     sequences: Union[str, List[str]] = Field(
         description="The input sequences to generate the text from.",
     )
-    num_generated_predictions: int = Field(
-        default=1,
-        description="The number of text generations to create from a single prompt. If "
-        "the same sequence is given as an input multiple times, the number of generated"
-        "the number of generated predictins is equivalent to the number of times the "
-        "the sequence is repeated.",
-    )
-    max_tokens: int = Field(
-        default=1024,
-        description="Maximum number of tokens to generate per output sequence. If no "
-        "value is provided, will default to 1024.",
-    )
-    return_logits: bool = Field(
-        default=False,
-        description="A flag that indicates whether to return "
-        "the logits for the input text sequence and the "
-        "generated text sequence. ",
-    )
     include_prompt_logits: bool = Field(
         default=False,
         description="A flag that indicates whether to return "
         "the logits for the prompt. If set, prompt_logits are "
         "`prepended` to the logits for the generated text sequence."
-        "Note: This flag is only applicable when return_logits "
+        "Note: This flag is only applicable when output_scores "
         "is `True`.",
     )
+    # TODO: Session ID vs session?
     session_id: Optional[str] = Field(
         default=None,
         description="A user may set a string identifier "
         "for the kv cache session. If None, "
         "and the model is using kv cache, it "
         "will be set to a random uuid.",
     )
+    # TODO: how does the padding length work with our two options of max and largest?
```

Comment:
if we can't support the transformers spec for padding length due to shape requirements, we should just raise an error if user tries to override in the generation config

Reply:
Not sure if that's necessary, just because `pad_token_id` is the only padding argument provided in the GenerationConfig, which serves a different purpose.